<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsClass</class>
 <widget class="QWidget" name="QtWidgetsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>91</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>91</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>91</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsClass</string>
  </property>
  <widget class="QPushButton" name="recButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/buttons/rec-icon-png-23.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sizeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/buttons/crop.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="wholeScreenButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/buttons/stop.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseResumeButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/buttons/playpause.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pathButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose file path</string>
   </property>
  </widget>
  <widget class="QLabel" name="pathText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Record sound</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="openPath">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Open folder</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Buttons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>recButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>on_RECButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>on_STOPButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseResumeButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>on_PAUSEButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>on_RESIZEButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>on_PATHButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openPath</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>on_OPENPATHButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wholeScreenButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>on_FULLSCREENButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CreateThreads()</slot>
  <slot>on_RECButton_clicked()</slot>
  <slot>on_STOPButton_clicked()</slot>
  <slot>on_PAUSEButton_clicked()</slot>
  <slot>on_RESIZEButton_clicked()</slot>
  <slot>on_FULLSCREENButton_clicked()</slot>
  <slot>on_PATHButton_clicked()</slot>
  <slot>on_OPENPATHButton_clicked()</slot>
 </slots>
</ui>
